.github/workflows/pr-check.yml

```
@@ -0,0 +1,125 @@
+        run: npm ci
+
+      - name: Build project
+        run: npm run build
+
+      - name: Audit with Lighthouse
+        id: lighthouse
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          urls: |
+            https://felhosip-web.github.io/Cica-NyT/
+          uploadTargets: temporary-public-storage
```

actionlint (1.7.12)

[error] input "uploadTargets" is not defined in action "treosh/lighthouse-ci-action@v12". available inputs are "artifactName", "basicAuthPassword", "basicAuthUsername", "budgetPath", "configPath", "runs", "serverBaseUrl", "serverToken", "temporaryPublicStorage", "uploadArtifacts", "uploadExtraArgs", "urls"

(action)